data/desktop/Makefile

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+BINDIR := /usr/bin
+DATADIR := /usr/share
+
+SOURCES := $(wildcard *.in)
```

Comment:
Nitpick, I'm a bit allergic to the wildcard function as it is easy to use incorrectly. Can you please list the file explicitly here.